CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1736**
+  - The `@cumulus/discover-granules` task now sets the `dataType` of discovered
+    granules based on the `name` of the configured collection, not the
+    `dataType`.
+  - The config schema of the `@cumulus/discover-granules` task now requires that
+    collections contain a `version`.
+  - The `@cumulus/sync-granule` task now sets the `dataType` of granules based
```

Comment:
This is in some case where `granule.dataType` was not set during the discovery?

Reply:
I'll update it to read:
```md
- The `@cumulus/sync-granule` task will set the `dataType` and `version` of a
    granule based on the configured collection if those fields are not already
    set on the granule. Previously it was using the `dataType` field of the
    configured collection, then falling back to the `name` field of the
    collection. This update will just use the `name` field of the collection to
    set the `dataType` field of the granule.
```

Reply:
Looking at the the `SyncGranule` code, it's very confusingly written. I'll write another ticket to take a closer look at that and make sure that it's behaving as expected. This PR doesn't substantively change that behavior, though, so I think it should be tackled in another PR.

Reply:
Created CUMULUS-1789 for this.